prow/cmd/deck/static/pr/pr.ts

```diff
@@ -134,7 +134,7 @@ function getPRQuery(q: string): string {
             || tkn === "type:pr");
     }).join(" ");
     // Returns the query with is:pr to the start of the query
-    result = "is:pr " + result;
+    result = "is:pr -is:draft" + result;
```

Comment:
I don't think hiding draft PRs from the PR status page is the right approach. Can we instead indicate that the PR cannot be merged because it is a draft?

Reply:
Also this token needs to be added to the list on line 133.

Reply:
We could, or if we get the WIP-label change in we can just not change anyhting here

Reply:
Tide needs an independent solution that doesn't rely on the WIP plugin in case it is not enabled on the repo.

Reply:
It would be nice if Tide could assume some plugins like `wip`, and `hold` are enabled, but I don't know of nice way to do that.
Automatically (and transparently) enabling plugins for repos handled by Tide or requiring that the plugins are enabled in order to use Tide both sound like bad ideas to me.

Reply:
IMO the goal should be for the search to exclude as many things as possible, to keep the search as cheap as possible.

Long term I would like to see us reevaluate and change the current design of writing an update to date "here is why it cannot merge" status on each PR. A simpler, cheaper design would be to only toggle between not ready / can merge. Clicking the detail link can then bring you to a page that will fetch the PR and calculate the various reasons why it cannot merge.

Reply:
I agree except with the part about our goal being "exclude as many things as possible, to keep the search as cheap as possible".  That is potentially true for Tide if adding more constraints doesn't make the query more likely to fail, but I don't think we care about search cost at all for Deck's PR status page which is what we are discussing here.